library/lua/gui.lua

```diff
@@ -818,10 +820,14 @@ function ZScreen:onInput(keys)
         end
         local passit = self.pass_pause and keys.D_PAUSE
         if not passit and self.pass_mouse_clicks then
-            for key in pairs(MOUSE_KEYS) do
-                if keys[key] then
-                    passit = true
-                    break
+            if keys.CONTEXT_SCROLL_UP or keys.CONTEXT_SCROLL_DOWN then
```

Comment:
Does this also trigger if you press the keys bound to these bindings?

Reply:
if there are keyboard keys bound to these interface keys, then yes. these interface keys are normally bound to the mouse scroll wheel, though.

Reply:
![image](https://github.com/DFHack/dfhack/assets/977482/7b49838c-3991-4613-87d4-97d4acd40a9a)